test/k8sT/manifests/iperf3-deployment.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: iperf3-server
+  labels:
+    id: iperf3-server
+    zgroup: testapp
+spec:
+  hostNetwork: true
+  terminationGracePeriodSeconds: 0
+  containers:
+    - name: iperf
+      image: docker.io/cilium/netperf:2.0
+      command: ["iperf3", "-s"]
+  nodeName: k8s2
```

Comment:
That will fail on GKE & EKS. Can we use a DaemonSet instead?

Reply:
Just to clarify, is it because of the `nodeName`? If so, this problem is related to the client as well. The idea is to force the deployment of the client and sever in a different nodes. If I use a DaemonSet, the client has to use the IP of the server that is in the another node. So, I have to check how I can do that.

Reply:
> Just to clarify, is it because of the nodeName?

Yes.

> The idea is to force the deployment of the client and sever in a different nodes.

Using a DaemonSet will do that.

> If we use a DaemonSet, the client has to use the IP of the server that is in the another node. So, I have to check how I can do that.

Maybe check `fetchPodsWithOffset`. We might have a different helper for that purpose in Services.go.